pkg/controller/computenodeopenstack/computenodeopenstack_controller.go

```diff
@@ -87,7 +92,16 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
-	err = c.Watch(&source.Kind{Type: &corev1.Pod{}}, &handler.EnqueueRequestForOwner{
+	err = c.Watch(&source.Kind{Type: &batchv1.Job{}}, &handler.EnqueueRequestForOwner{
+		IsController: true,
+		OwnerType:    &computenodev1alpha1.ComputeNodeOpenStack{},
+	})
+	if err != nil {
+		return err
+	}
+
+	// Watch ConfigMaps owned by ComputeNodeOpenStack
+	err = c.Watch(&source.Kind{Type: &corev1.ConfigMap{}}, &handler.EnqueueRequestForOwner{
```

Comment:
not sure if we need to reconcile on this, but we can remove it later if not needed